dotcom-rendering/src/web/components/TableOfContents.importable.tsx

```diff
@@ -22,25 +22,16 @@ const anchorStyles = (palette: Palette) => css`
 	padding: ${space[1]}px 0 ${space[4]}px 0;
 `;
 
-const defaultListItemStyles = (palette: Palette) => css`
-	border-top: 1px solid ${line.primary};
-	&:hover {
-		border-top: 1px solid ${palette.text.tableOfContents};
-		cursor: pointer;
-	}
-`;
-
 const listItemStyles = (format: ArticleFormat, palette: Palette) => {
-	if (format.display === ArticleDisplay.Immersive) {
-		return css`
-			${headline.xxxsmall({ fontWeight: 'light' })}
-			${defaultListItemStyles(palette)}
-		`;
-	}
-
+	const fontWeight =
+		format.display === ArticleDisplay.Immersive ? 'light' : 'bold';
 	return css`
-		${headline.xxxsmall({ fontWeight: 'bold' })}
-		${defaultListItemStyles(palette)}
+		${headline.xxxsmall({ fontWeight })};
+		border-top: 1px solid ${line.primary};
+		&:hover {
+			border-top: 1px solid ${palette.text.tableOfContents};
```

Comment:
Yea that's right. The final design is to also make this line thicker, but because it was not a straight forward change, we agreed that for now we leave it as it is and work on the tickness after release. Do you still think it may be confusing, even with the thicker design?
![image](https://user-images.githubusercontent.com/15894063/212862109-4e52e5ba-1ad3-4032-9037-e3d0ad7f0c71.png)

Reply:
Yeaa I see your point. @HarryFischer what do you think?